src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -183,10 +183,12 @@ HttpStatusCode mongoSubscribeContext
     {
       std::string oidString = oid.toString();
 
-      orion::subCache->insert(new orion::Subscription(tenant, servicePath, requestP, oidString, expiration, notifyFormat));
+      LM_T(LmtMongoSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
+      mongoSubCacheItemInsert(tenant.c_str(), servicePath.c_str(), requestP, oidString.c_str(), expiration, throttling, notifyFormat);
     }
 
 
+    reqSemGive(__FUNCTION__, "ngsi10 subscribe request", reqSemTaken);
```

Comment:
Your or not, what we should check in this PR is if this (and others) semGive movements make sense. What do you think in this particular case? For me, it could make sense.